utils/prometheus/Makefile

```diff
@@ -4,6 +4,7 @@ PKG_NAME:=prometheus
 PKG_VERSION:=2.15.2
 PKG_RELEASE:=1
 
+
```

Comment:
Fantastic idea!

Now please add another pull request to something I don't own.